sig-release/README.md

```diff
@@ -0,0 +1,122 @@
+- Set and enforce criteria for repository inclusion in a Kubernetes release
+  - Governance
+  - Stabilization
+  - Test coverage
+- Set and enforce criteria for code inclusion in a Kubernetes release
+  - Feature flags
+  - Documentation
+  - Test coverage
+  - Dashboards
+  - Status reports
+- Define template and format release communications
+- Deriving signal from the following sources
+  - [test grid](https://k8s-testgrid.appspot.com/)
+  - GitHub [flake issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Akind%2Fflake)
+  - GitHub [bug issues](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aopen%20is%3Aissue%20label%3Akind%2Fbug)
+- Resolve release blocking issues
+- Identifying owning individuals and SIGs for blocking issues
+- Working with SIGs and individuals to drive resolution of open issues
+- Building summary of release criteria and status and publish to the community on a regular basis throughout the release cycle
+
+## Relation to the Release Manager and Team
+
+The Kubernetes Release Team is embedded within SIG Release and is responsible for the day to day work required to
+successfully release while the SIG at large is focused on the continued improvement of the release process. Historically
+the Release Manager (previously Release Czar)
+and later Release Team has assumed the following responsibilities
+- Authority to build and publish releases at the published release date under the auspices of the CNCF
+- Authority to accept or reject cherrypick merge requests to the release branch
+- Authority to accept or reject PRs to the kubernetes/kubernetes master branch during code slush period
+- Changing the release timeline as needed if keeping it would materially impact the stability or quality of the release
+
+these responsibilities will continue to be discharged by SIG release through the Release Team. This charter grants SIG 
+Release the following additional responsibilities
+- Authority to revert code changes which imperil the ability to produce a release by the communicated date or otherwise 
+  negatively impact the quality of the release
+- Authority to guard code changes behind a feature flag which do not meet criteria for release
+- Authority to close the submit queue to any changes which do not target the release as part of enforcing the code slush
+  period
+
+which shall be the discharged by the Release Team.
+
+## Ongoing responsibilities of the Release Team
+The release team will be responsible for the following items, as well as remaining active members of SIG-release for the
+duration of the release.
+
+### "Minor" Releases 
+For each minor release (e.g. 1.7, 1.8)
+- Shepherd each release to completion
+- Define and enforce release criteria
+- Test health
+- GitHub issues
+- Define and enforce development process as it relates to the release
+  - Code slush time and Merge Queue access
+  - Enforcing feature freeze + approving exceptions
```

Comment:
Feature freeze enforcement has to be performed by a release team, together with SIG-PM.